src/types.tsx

```diff
@@ -0,0 +1,46 @@
+export interface State {
```

Comment:
Do you think we should change the name to interfaces.tsx? This is more of a question!